src/jq.erl

```diff
@@ -21,7 +29,14 @@ init() ->
         Dir ->
             filename:join(Dir, ?LIBNAME)
     end,
-    erlang:load_nif(SoName, 0).
+    %% Start the jq application here since it needs to be started so
+    %% we can read its properties
+    application:start(jq),
```

Comment:
Small nit: maybe `application:load` is enough.

<details>

<summary>  Quick test in emqx: </summary>

```erlang
2> application:get_all_env(gen_rpc).
[]
3> application:load(gen_rpc).
ok
4> application:get_all_env(gen_rpc).
[{socket_keepalive_interval,5},
 {call_receive_timeout,15000},
 {rpc_module_list,[]},
 {socket_keepalive_idle,5},
 {client_config_per_node,{internal,#{}}},
 {async_call_inactivity_timeout,infinity},
 {send_timeout,5000},
 {connect_timeout,5000},
 {sbcast_receive_timeout,15000},
 {ssl_client_port,5370},
 {default_client_driver,tcp},
 {tcp_client_port,5369},
 {rpc_module_control,disabled},
 {authentication_timeout,5000},
 {client_inactivity_timeout,infinity},
 {ssl_server_port,false},
 {ssl_server_options,[]},
 {ssl_client_options,[]},
 {socket_keepalive_count,2},
 {tcp_server_port,5369},
 {server_inactivity_timeout,infinity}]
```

</details>

Reply:
Thansk, seems to work (have fixed in f6f295b)